lib/request-proxy.js

```diff
@@ -122,6 +149,77 @@ proto.proxyReq = function proxyReq(opts) {
         });
         res.end(res2);
     }
+
+    function sendHandler() {
+        // Per-request parameters override instance-level
+        maxRetries = numOrDefault(maxRetries, self.maxRetries);
+        retrySchedule = retrySchedule || self.retrySchedule;
+
+        var numRetries = 0;
+        var reqStartTime = Date.now();
+        var retryStartTime = null;
+
+        return function sendIt(options, head, body, callback) {
+            if (self.maxRetries === 0) {
+                ringpop.channel.send(options, '/proxy/req',
+                    head, body, callback);
+            } else {
```

Comment:
suggestion: can reduce indentation by early returning in `if` case and not using `else`.